services/src/main/java/org/keycloak/organization/utils/Organizations.java

```diff
@@ -229,7 +229,7 @@ public static OrganizationModel resolveOrganization(KeycloakSession session, Use
         }
 
         organization = ofNullable(user).stream().flatMap(provider::getByMember)
-                .filter(o -> o.isEnabled() && provider.isManagedMember(o, user))
```

Comment:
@olivierboudet I'm wondering if we can revert this change. The reasons are:

* The idea behind this check is to make sure that `unmanaged` members should not automatically resolve to the organization they are a member because they might choose to authenticate using their realm credentials. Differently than `managed` members given that the source of truth for their identities is either a broker associated with the organization or the organization itself if the user was created in the context of an organization (e.g.: registration link).

Even though we did not have test failures when changing this behavior, I think we should keep the logic as is.

Wdyt?

Reply:
not sure about that, here are my doubts :
* Even if organization is resolved automatically for `unmanaged` members, they should be allowed to authenticate with username/password because `Organization.tryRedirectBroker` do not redirect automatically is user has local credential
*  my need is specifically to resolve organization a user is member of to be able to display its identity providers. with this check, an `unmanaged` member of an organization without a matching domain email will never be able to use the organization IDP

do I miss someting ?

Reply:
Makes sense. Thanks.